hack/test-end-to-end.sh

```diff
@@ -105,6 +105,9 @@ function cleanup()
 	echo
 
 	if [[ -z "${SKIP_TEARDOWN-}" ]]; then
+		echo "[INFO] Switch back to 'default' project with 'admin' user for cleanup"
```

Comment:
Is it worth doing a `-n` check on $CLUSTER_ADMIN_CONTEXT? If you start e2e and hit ctrl-c fast enough, it will be empty.